src/components/file-uploader/UIFile.vue

```diff
@@ -17,7 +17,7 @@
                         @click="$emit('removed', file)"
                         v-html="clearIcon" />
                 <button v-else-if="failedToUpload"
-                        class="transition hover:rotate-180 !duration-200"
+                        class="transition-transform hover:-rotate-180 duration-300"
                         @click="uploadFile"
                         v-html="retryIcon" />
```

Comment:
Need to devise a design for when the user wants to remove a file that refuses to upload.